deployment/helm/node-feature-discovery/values.yaml

```diff
@@ -450,7 +450,7 @@ topologyUpdater:
   affinity: {}
   podSetFingerprint: true
 
-topologyGC:
+gc:
   enable: true
```

Comment:
I Think we should tie this to `enableNodeFeatureApi: false` (line 13 of the same file) 
why having it set to `true` if currently default `enableNodeFeatureApi` is false?

Reply:
in the `nfd-gc.yaml` we could 
```yaml
{{- if and .Values.enableNodeFeatureApi (or .Values.topologyUpdater.enable) -}}
```

something like that, could work?.....